code_snippets/Deploy_Previously_Created_Model.ipynb

```diff
@@ -0,0 +1,159 @@
+{
```

Comment:
<p>console - typo</p><p> </p><p>This is interesting... what about a programmatic way to list the models available?</p><p> </p><p>Can we use the SM PySDK instead, or is boto3 the only interface for what you're doing here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1479/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
It for sure would be possible to programmatically list models available. I did not want to do that in this code snippet since I think it is out of scope for focusing on deployment.

The approach I used for deploying is based on the one the AWS documentation uses here: https://docs.aws.amazon.com/sagemaker/latest/dg/ex1-deploy-model.html